Source/WebCore/platform/wpe/PlatformScreenWPE.cpp

```diff
@@ -94,17 +94,22 @@ bool screenHasInvertedColors()
     return false;
 }
 
-double screenDPI()
+double screenDPI(PlatformDisplayID screendisplayID)
 {
 #if ENABLE(WPE_PLATFORM)
-    auto* data = screenData(primaryScreenDisplayID());
+    auto* data = screenData(screendisplayID);
     return data ? data->dpi : 96.;
 #else
     notImplemented();
     return 96;
 #endif
 }
 
+double fontDPI()
+{
+    return screenDPI(primaryScreenDisplayID()); // No notion of separate font scaling in WPE
```

Comment:
We have a code style rule that comments are expected to be complete sentences. We also usually put the comment on the line above rather than on the same line.

Reply:
Changed comment to `// In WPE, there is no notion of font scaling separate from device DPI.` and moved to the previous line.